src/styles/getNavigationModalCardStyles/index.desktop.ts

```diff
@@ -1,10 +1,16 @@
-export default () => ({
+import positioning from '../utilities/positioning';
+import GetNavigationModalCardStyles from './types';
+
+const getNavigationModalCardStyles: GetNavigationModalCardStyles = () => ({
```

Comment:
I realized that for several files, you created a named function by changing the original code that used to export named function.

Is this something that lint complains or is this something you changed voluntarily?

Reply:
This is due to TS style guide [here](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#file-organization)

Reply:
So `getNavigationModalCardStyles` exports 3 env specific files for web, desktop and native.
In types.ts for this module we defined functions which are common for each of the env specific files.
Fabio converted this function to be named exported because you can't type the definition of  default exported function but of course you can import things.
I hope I explained it well 😅